pkg/runtime/runtime.go

```diff
@@ -1948,7 +1948,7 @@ func (a *DaprRuntime) GetPubSub(pubsubName string) pubsub.PubSub {
 }
 
 func (a *DaprRuntime) isPubSubOperationAllowed(pubsubName string, topic string, scopedTopics []string) bool {
-	inAllowedTopics := false
+	inAllowedTopics, inProtectedTopics := false, false
```

Comment:
Nit, for readability:

```suggestion
	var inAllowedTopics, inProtectedTopics bool
```

Reply:
That does read better, fixed in c0718be3